src/olympia/reviewers/serializers.py

```diff
@@ -522,10 +522,8 @@ class DraftCommentSerializer(serializers.ModelSerializer):
     user = SplitField(
         serializers.PrimaryKeyRelatedField(queryset=UserProfile.objects.all()),
         BaseUserSerializer())
-    version = SplitField(
-        serializers.PrimaryKeyRelatedField(
-            queryset=Version.unfiltered.all()),
-        AddonBrowseVersionSerializer())
+    version = serializers.PrimaryKeyRelatedField(
+        queryset=Version.unfiltered.all())
```

Comment:
you can do `version_id = serializers.PrimaryKeyRelatedField(queryset=Version.unfiltered.all(), source='version')` and then you don't have to override it in `to_representation`  (`'version_id': self.get_version_object().pk` in `ReviewAddonVersionDraftCommentViewSet.get_extra_comment_data`